### PR TITLE
Version update for azure-core, azure-identity and eventhub

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Changes to update version or released packages:

azure-core to 1.1.2
azure-identity to 1.1.1
azure-eventhub to 5.0.0b7
azure-eventhub-checkpointstoreblob to 1.0.0b7
azure-eventhub-checkpointstoreblob-aio to 1.0.0b7

### CODE REVIEW

File: sdk/identity/azure-identity/azure/identity/_version.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-VERSION = "1.1.0"
+VERSION = '1.1.1'
```
Comment (chlowell) [inline sdk/identity/azure-identity/azure/identity/_version.py]: Please keep the double quotes so our formatting tool (black) doesn't revert the change.

Comment (praveenkuttappan) [reply sdk/identity/azure-identity/azure/identity/_version.py]: @chlowell  Done. Version was updated by version increment tool. I have fixed it manually for now. will submit another PR to fix this in tool.

File: sdk/core/azure-core/HISTORY.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 # Release History
 
 -------------------
+## Version 1.1.2 Unreleased
```
Comment (weshaggard) [inline sdk/core/azure-core/HISTORY.md]: Did you do these version updates manually or use the scripts? Does the script add the "Unreleased" keyword to the heading?

Comment (weshaggard) [reply sdk/core/azure-core/HISTORY.md]: Do we have docs that instruct folks how to update versions with the scripts?

Comment (praveenkuttappan) [reply sdk/core/azure-core/HISTORY.md]: Change log is updated manually. I have an issue pending with me to include change log update also in versioning tool. I will update the documentation on how to use the tool once this change to update change log is also completed
